src/SprykerEco/Client/ComputopShipment/QuoteShipmentExpander/QuoteDefaultShipmentExpander.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Client\ComputopShipment\QuoteShipmentExpander;
```

Comment:
Not a blocker. But QuoteExpander folder and QuoteExpanderInterface are more generic and can be used for more cases in the future instead of having very concrete one